convert.sh

```
@@ -540,7 +540,12 @@ for (( i = 0; i < tpmCount; i++ )); do
         comment+="; dma -1 in config (input only)"
     fi
 
-    echo "    DEF_TIM(${timchOut}, ${pin}, ${timUse}, 0, ${dopt}), // ${comment}" >> ${cFile}
+    # H7 DEF_TIM requires a 7th argument (upopt: TIM_UP burst DShot stream, NONE = burst disabled)
+    if [[ "$mcu" == STM32H7* ]]; then
+        echo "    DEF_TIM(${timchOut}, ${pin}, ${timUse}, 0, ${dopt}, NONE), // ${comment}" >> ${cFile}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)